src/applications/mhv-secure-messaging/components/shared/AlertBackgroundBox.jsx

```diff
@@ -188,7 +188,10 @@ const AlertBackgroundBox = props => {
   return (
     <>
       {activeAlert &&
-        activeAlert.header !== Alerts.Headers.HIDE_ALERT && (
+        activeAlert.header !== Alerts.Headers.HIDE_ALERT &&
+        activeAlert.content !==
+          (Alerts.Message.ATTACHMENT_SCAN_FAIL ||
+            Alerts.Message.MULTIPLE_ATTACHMENTS_SCAN_FAIL) && (

```

Comment:
I think we need to try another approach here. Instead of changing the logic within `AlertBackgroundBox` which will no longer be applicable to other scenarios except for scan fail - try changing `adding `Alerts.Headers.HIDE_ALERT` to `addAlert` in `sendReply` action where this error is caught